source/components/triggering-changes-with-actions.md

```diff
@@ -156,3 +156,56 @@ a function instead.
 Actions in components allow you to
 decouple an event happening from how it's handled, leading to modular,
 more reusable components.
+
+## Calling Actions Across Component Layers
+
+When your components go multiple layers deep, its common to need to handle an action several layers up the tree.
```

Comment:
I think it's important to differentiate between being nested in the context/template sense, and using block components and being nested in the markup sense. This section not being applicable to the latter.